junit-platform-launcher/src/main/java/org/junit/platform/launcher/TestPlan.java

```diff
@@ -101,11 +102,11 @@ protected TestPlan(boolean containsTests) {
 	@API(status = DEPRECATED, since = "1.4")
 	public void add(TestIdentifier testIdentifier) {
 		Preconditions.notNull(testIdentifier, "testIdentifier must not be null");
-		allIdentifiers.put(testIdentifier.getUniqueId(), testIdentifier);
-		if (testIdentifier.getParentId().isPresent()) {
-			String parentId = testIdentifier.getParentId().get();
+		allIdentifiers.put(testIdentifier.getUniqueIdObject(), testIdentifier);
+		if (testIdentifier.getParentIdObject().isPresent()) {
+			UniqueId parentId = testIdentifier.getParentIdObject().get();
 			Set<TestIdentifier> directChildren = children.computeIfAbsent(parentId,
-				key -> Collections.synchronizedSet(new LinkedHashSet<>(16)));
+				key -> synchronizedSet(new LinkedHashSet<>(16)));
```

Comment:
This has not changed, but how is this value of 16 chosen?

Reply:
Probably just a wild guess that assumes most test classes have more than 8 children.